bindings/rust/src/lib.rs

```diff
@@ -153,12 +151,6 @@ impl ExecutionResult {
     }
 }
 
-impl From<ExecutionResult> for sys::FizzyExecutionResult {
```

Comment:
This has not been used and probably will not be used in this form, rather as a `Box<>`. Will add it back when needed.